cvat/apps/dataset_manager/bindings.py

```diff
@@ -908,7 +908,7 @@ def _make_image(i, **kwargs):
                         attributes["labels"].append({"label_id": idx, "name": label["name"], "color": label["color"]})
                         attributes["track_id"] = -1
 
-                dm_item = datumaro.DatasetItem(id=osp.split(frame_data.name)[-1].split('.')[0],
+                dm_item = datumaro.DatasetItem(id=osp.splitext(osp.split(frame_data.name)[-1])[0],
```

Comment:
Probably, this should not be in this PR.

Reply:
Okay, I'll made these changes in new PR